modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/_form-builder-drag-and-drop.scss

```diff
@@ -85,10 +85,6 @@
 	}
 }
 
-div.sortable-layout-drag-indicator {
-	background-color: rgba(17, 17, 17, 0);
```

Comment:
Hey @jonmak08 we need this line declaration, because it overwrites the default style of the AlloyUI when an user drags a field in form builder, but I think we can change from`background-color: rgba(17, 17, 17, 0);` to `background-color: transparent;` because we just need to hide the default line of the AlloyUI in the form builder.